tests/viadot_tests/integration/test_sql_server.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from datetime import datetime, timedelta, timezone
+import struct
+
+from viadot.sources import SQLServer 
+
+
+@pytest.fixture
+def sql_server():
```

Comment:
Please also add test to `exists()` method. We also don't have any unit test. If it's possibly, please add it by mocking database.